playbook/app/pb_kits/playbook/pb_filter/docs/_filter_default.html.erb

```diff
@@ -30,7 +30,11 @@
     <%= form.collection_select :example_collection_select, example_collection, :value, :name, props: { label: true } %>
 
     <%= form.actions do |action| %>
-      <%= action.submit props: { text: "Apply", data: { disable_with: "<i class='far fa-spinner fa-spin mr-3'></i>Searching...".html_safe },}%>
+      <%= action.submit props: {
+        text: "Apply",
+        data: {
+          disable_with: "<i>pb_rails('icon', props: { icon: 'spinner', spin: true, fixed_width: true })</i>Searching...".html_safe
```

Comment:
@skduncan 
1) I think this is wrong here. We shouldn't be wrapping pb_rails with an `<i>` tag. 
2) I tested in milano environment, and the spinner is not showing up. To be fair, production of the playbook-website isn't working either (it's not showing a spinner). This should be tested in nitro with an alpha on a rails form that is using the Filter kit, so we can make sure this works in nitro as we expect.